hdat/hdat_cli.py

```diff
@@ -38,6 +39,18 @@ def parse_arguments(arguments):
     verify_result_help = 'results to be stripped and moved into the golden store'
     verify_parser.add_argument('resultspec', nargs='?', default='', metavar='<result>', help=verify_result_help)
 
+    # we should think through the command line API for this; we should try to
```

Comment:
This type of comment is better suited for a comment on a Github issue, given that we have it set up and are actively using the issue tracker for this project (other codebases could be different). 

To your point: there are many different ways, pytest for example lets you put in logical operators like ""MyClass and not method". The most common analog would really be selecting files (which is handled by the shell and includes `*`, `?`, and other things like `[0-9]` for selecting a number)